### DISCUSSION TITLE
New List Keeps Loading

### DISCUSSION THREAD

**Discussion by elanizi43998** [community/community#61757] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

# Hello everyone 👋

I've noticed that when I want to create a new list It keeps loading after Create Is clicked, and seems to be stuck in the UI until I close It myself.

### PS: The list is created even if I close the window while the button in disabled and loading.

<img width="1295" alt="Screenshot 2023-07-25 at 15 58 52" src="https://github.com/community/community/assets/98546093/a26ba29d-3ca9-462f-8316-7c1faf1cf767">

**Comment by moonfury-ops** [comment]
Thank you

**Reply by mikuldeejay** [reply]
Cool

**Reply by Marcysiegel** [reply]
Hi 👋

**Reply by alialali313** [reply]
https://cmujtucpv4/-231029_0656.pdf

**Reply by alialali313** [reply]
8931 0101 5191 https://youtu.be/yTcuazSy5Cs
aljbore40004@gmail.com

**Reply by alialali313** [reply]
"kv18": "1523297725",
      "adultRisk": "high",
      "impressions": 416

**Reply by alialali313** [reply]
http://apps.apple.com/<country>/app/<app–name>/id<store-ID>

**Reply by Marcysiegel** [reply]
Please who are you and where are you from?

**Comment by aldoyh** [comment]
Here I am facing the exact same thing.

I took screenshots hoping nothing sensitive shown!

![Screenshot 2023-09-15 at 11 45 17](https://github.com/community/community/assets/5215449/382b0c66-ca2f-45af-b6ad-14b4eb1fb4b2)

**Reply by mikuldeejay** [reply]
🙋

**Reply by xcichyX** [reply]
> Here I am facing the exact same thing.
> 
> I took screenshots hoping nothing sensitive shown!
> 
> ![Screenshot 2023-09-15 at 11 45 17](https://github.com/community/community/assets/5215449/382b0c66-ca2f-45af-b6ad-14b4eb1fb4b2)
>

**Comment by savioti** [comment]
same problem here, but the list is created when I try to add a repo to it in another tab

**Reply by Blaine2369** [reply]
I really have no idea what the fuck I'm doing

**Reply by UniversalGatekeeper** [reply]
UniversalGatekeeper of AI
Sometimes Not Knowin What Ur Doin is Phenomenal .... Thats Wat Were Here 4.

**Reply by Marcysiegel** [reply]
Hello my name is marcy and you?

**Comment by Salaheldean** [comment]
Hi

**Comment by Enes0120** [comment]
Hi

**Reply by UniversalGatekeeper** [reply]
UniversalGatekeeper of AI
Phenomenal

**Reply by UniversalGatekeeper** [reply]
> Hi

^Hello

**Comment by ossanche** [comment]
DiXAO #1

**Reply by DWMMI** [reply]
oxcar

**Reply by Marcysiegel** [reply]
> oxcar
> 

Hello my name is marcy and you?

**Reply by DWMMI** [reply]
I'm Marcos nice to meet you Marcy, where are U from ?

**Comment by mikuldeejay** [comment]
Let's do it

**Reply by Marcysiegel** [reply]
> Let's do it
> 
> On Sat, Nov 4, 2023, 6:57 PM Marcysiegel ***@***.***> wrote:
> 
> > Hello my name is marcy and you?
> >
> > —
> > Reply to this email directly, view it on GitHub
> > <https://github.com/orgs/community/discussions/61757#discussioncomment-7474998>,
> > or unsubscribe
> > <https://github.com/notifications/unsubscribe-auth/BCBEWJP2FTXEIGKN7QPJWXTYCZQYNAVCNFSM6AAAAAA2XHBIGSVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM3TINZUHE4TQ>
> > .
> > You are receiving this because you commented.Message ID:
> > ***@***.***>
> >
> 

Hello my name is marcy and you?

**Comment by mikuldeejay** [comment]
Michael

**Comment by mikuldeejay** [comment]
I'm good, how about you?

**Reply by Marcysiegel** [reply]
Please where are you from?

**Reply by Marcysiegel** [reply]
Can I no you

**Comment by Jamim** [comment]
This discussion duplicates #50880, just as a dozen of other reports.

**Comment by mikuldeejay** [comment]
NAIROBI KENYA

**Comment by saugup-ms** [comment]
Good
